doc/source/guide/acquiring_data/database.rst

```diff
@@ -337,7 +317,7 @@ values as keyword arguments to :class:`tables.DatabaseEntry` as follows:
     >>> entry in database
     True
     >>> len(database)
-    30
+    10
```

Comment:
why are all changed to `10`? either you've run the queries and the data was not updated (maybe you didn't have connection to internet?) or there's something broken in the `database` itself...